### PR TITLE
clamav: add new package

### PR DESCRIPTION
**Pull request by davydden**

### CODE REVIEW

File: var/spack/repos/builtin/packages/clamav/package.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    depends_on('zlib')
+    depends_on('bzip2')
+
+    def configure_args(self):
+        spec = self.spec
+        args = [
+            '--disable-dependency-tracking',
+            '--disable-silent-rules',
+            #--libdir=#{lib}
+            #--sysconfdir=#{etc}/clamav
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/clamav/package.py]: These lines are causing the flake8 tests to fail.

File: var/spack/repos/builtin/packages/clamav/package.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            '--disable-zlib-vcheck',
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/clamav/package.py]: Any idea why this is necessary? Based on the above flags, I have a feeling you took a Homebrew recipe and ported it to Spack.

Does anyone know what things like `--disable-dependency-tracking` and `--disable-silent-rules` actually do?? Every Homebrew package uses them, but most Spack packages don't. If we determine that they are useful, we could add them as default arguments in the `AutotoolsPackage` base class.

Comment (davydden) [reply var/spack/repos/builtin/packages/clamav/package.py]: yes, i just ported what Homebrew does and made sure it picks up all dependencies.

No idea what those `--disable-xyz` do.

**Comment by citibeth** [comment]
I'm a little hesitant to merge given the unanswered questions.

**Comment by davydden** [comment]
> I'm a little hesitant to merge given the unanswered questions.

I removed those lines, package still builds ok on macOS.

**Pull request merged**